Makefile

```diff
@@ -20,7 +23,9 @@ test-ci:
 	@NODE_ENV=test NODE_PATH=lib XUNIT_FILE=$(CIRCLE_TEST_REPORTS)/mocha.xml \
 	./node_modules/.bin/istanbul cover \
 	./node_modules/mocha/bin/_mocha --report lcovonly -- -R xunit-file
+ifdef CODECLIMATE_REPO_TOKEN
```

Comment:
the `test-ci` target is intended only to run in CI, so this envvar should always be present. `test-cov` is the local equivalent